pkgs/tools/misc/pastebinit/test.nix

```diff
@@ -0,0 +1,25 @@
+{ runCommand
+, pastebinit
+, cacert
+, wget
+}:
+runCommand "pastebinit-test" { nativeBuildInputs = [ pastebinit cacert wget ]; } ''
+  export USER="nixpkgs-pastebinit-test"
+  teststring="test from pastebinit"
+  for pastebin in $(pastebinit -l | sed 1d | cut -d' ' -f2); do
+    echo "Trying $pastebin"
```

Comment:
Hmm, did this work for you? I'm getting:
```
$ nix-build . -A pastebinit.tests
(...)
Trying dpaste.com
Failed to contact the server: <urlopen error [Errno -3] Temporary failure in name resolution>
error: builder for '/nix/store/gilyr1i9jz31mr6y8lx48vgmgvshlwaj-pastebinit-test.drv' failed with exit code 1;
```
As much as I like automated tests, perhaps we should skip the integration tests for now. Perhaps create a local end-to-end test as soon as someone packages a pastebin server for nixos? :)